bundler/bundler.go

```diff
@@ -426,6 +426,15 @@ func (b *Bundler) fetchIntermediates(certs []*x509.Certificate) (err error) {
 			// construct the filename as the CN with no period and space
 			name = strings.Replace(cert.Subject.CommonName, ".", "", -1)
 			name = strings.Replace(name, " ", "", -1)
+
+			// add SKI as extra identifier
+			// in case SKI is nil, use serial number as extra identifier
+			if cert.SubjectKeyId != nil {
+				name += fmt.Sprintf("_SKI_%x", cert.SubjectKeyId)
+			} else {
+				name += fmt.Sprintf("_serial_%x", cert.SerialNumber.Bytes())
```

Comment:
It might make sense to just ignore the serial number, and append a (truncated) hash of the `cert.Raw`.